PATENTS.md

```diff
@@ -1,5 +0,0 @@
-# FDC3 Patent Terms
```

Comment:
There is also a version of this in the charter document - however, I think we need it somewhere in the root of the project - either in the LICENSE file or NOTICES.md file (tack it onto the end after the license acceptance section for the CSL), particularly as these license terms still govern past contributions from participants who have not yet accepted the CSL.

I am aware that PATENTS-FDC3-1.0.md is not removed in this PR - but it's not covering 1.2-2.0.

Reply:
added to README